packages/rspack/src/Compiler.ts

```diff
@@ -163,12 +159,22 @@ class Compiler {
 			get EntryPlugin() {
 				return require("./builtin-plugin").EntryPlugin;
 			},
+			get ExternalsPlugin() {
+				return require("./builtin-plugin").ExternalsPlugin;
```

Comment:
This lazy valuation seems not working as `builtin-plugin` is just exporting everything.

Reply:
Not affect the correctness, we can make it lazy in following PR